script.js

```diff
@@ -0,0 +1,487 @@
+    if (index % 2 !== 0) {
+      return false;
+    }
+    if (replacedKeyItem === letter) {
+      return true;
+    }
+    return false;
+  });
+  if (positionInArray !== -1) {
+    return replacedKeysArr[positionInArray + 1];
+  }
+  return false;
+}
+
+function getMupkupFromLetter(letter) {
+  if (typeof letter === 'object') {
+    return `<span class="key">${letter[0]}</span>`;
+  }
+  if (letter === ' ') {
+    return `<span class="key extra-wide-key">${letter}</span>`;
+  }
+  if (letter === 'CapsLock') {
+    return `<span class="key caps-key">${letter}</span>`;
+  }
+  if (letter.indexOf('Arrow') !== -1) {
+    return `<span class='key-wrap'>
+            <span class="key hide-text">${letter}</span>
+            <span class="replacing-text arrow-key arrow-${getReplacingText(letter).toLowerCase()}">${getReplacingText(letter)}</span>
+            </span>`;
+  }
+  if (getReplacingText(letter) !== false) {
+    return `<span class='key-wrap'>
+            <span class="key hide-text">${letter}</span>
+            <span class="replacing-text">${getReplacingText(letter)}</span>
+            </span>`;
+  }
+  return `<span class="key">${letter}</span>`;
+}
+
+function addKeysToRow(row, lettersSet) {
+  const rowElem = row;
+  const rowKeysMurkup = lettersSet.map((letter) => getMupkupFromLetter(letter)).join('');
+  rowElem.innerHTML = rowKeysMurkup;
+}
+
+function getSubarrayForRow(lettersSet, rowIndex) {
+  let subarrayStartPosition = 0;
+  if (rowIndex > 0) {
+    subarrayStartPosition = ROWS_SIZE[rowIndex - 1];
+  }
+  const subarrayEndPosition = ROWS_SIZE[rowIndex];
+  return lettersSet.slice(subarrayStartPosition, subarrayEndPosition);
+}
+
+function fillWithLetters() {
+  const rows = templateElem.querySelectorAll('.row');
+  const lettersSet = lettersList[`${lang}Keys`];
+
+  rows.forEach((row, index) => addKeysToRow(row, getSubarrayForRow(lettersSet, index)));
+}
+
+function isRegularLetter(innerText) {
+  return innerText.length === 1;
+}
+
+function textTyping(innerText) {
+  const cursorPosition = textInput.selectionStart;
+  const textBeforeCursor = textInput.value.slice(0, cursorPosition);
+  const textAfterCursor = textInput.value.slice(textInput.selectionEnd);
+  textInput.value = textBeforeCursor + innerText + textAfterCursor;
+  textInput.selectionStart = cursorPosition + 1;
+  textInput.selectionEnd = cursorPosition + 1;
+}
+
+function isLangChangingPressed(pressedKeysArray) {
+  const pressedCodeList = pressedKeysArray.map((key) => key.innerText);
+  return LANG_CHANGING_KEYS[0].every((key) => pressedCodeList.indexOf(key) !== -1)
+  || LANG_CHANGING_KEYS[1].every((key) => pressedCodeList.indexOf(key) !== -1);
+}
+
+function deleteSelectedDiapason() {
+  textInput.value = textInput.value.slice(0, textInput.selectionStart)
+  + textInput.value.slice(textInput.selectionEnd);
+}
+
+function isMultiContentKey(index) {
+  if (index <= MULTI_CONTENT_KEYS) {
+    return true;
+  }
+  return false;
```

Comment:
You can use just _return index <= MULTI_CONTENT_KEYS_ insead off explicitly using return true and return false)